src/bloqade/qasm2/parse/lowering.py

```diff
@@ -27,6 +27,7 @@ def run(
         lineno_offset: int = 0,
         col_offset: int = 0,
         compactify: bool = True,
+        return_defs: bool = False,
```

Comment:
this is breaking the interface too. pyright would have errored here for you locally